templates/data-access-predict/README.md

```diff
@@ -0,0 +1,15 @@
+# Template: Data Access Predict
+
+This template shows Sema4.ai Data Access in action with predictive models. It contains all the necessary data files and follows the required structure to get started with building predictive models and creating `@predict` actions that add predictive capabilities to your agent.
+
+Follow the full tutorial using this template in our [documentation](https://sema4.ai/docs/solutions/data-access/models).
+
+## Predictive Models
+
+This example contains three scenarios:
+
+1. **Linear Regression** using the energy consumption data
+2. **Classification** using the customer classification data
```

Comment:
Do we need to drop point 2. as the data source is no there

Reply:
Changed this to be more clear.